perceval/backends/core/mediawiki.py

```diff
@@ -303,7 +325,17 @@ def fetch_all_pages(namespaces_contents):
                         apcontinue = None
                     pages_json = data_json['query']['allpages']
                     for page in pages_json:
-                        yield self.__get_page_reviews(page)
+                        if page['pageid'] in pages_done:
+                            # The page was already returned for previous revisions
+                            continue
+
+                        pages_done.append(page['pageid'])
+                        page_reviews = self.__get_page_reviews(page)
+                        if not page_reviews:
+                            # Page without reviews are not managed
```

Comment:
I think we need to add a warning message to inform this page was skipped. Like in https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/mbox.py#L213